app/connectors/ArrivalMovementConnector.scala

```diff
@@ -16,17 +16,25 @@
 
 package connectors
 
+import com.lucidchart.open.xtract.XmlReader
 import config.FrontendAppConfig
+import connectors.CustomHttpReads.rawHttpResponseHttpReads
 import javax.inject.Inject
-import models.{ArrivalId, Arrivals}
+import models.arrival.{MessagesSummary, XMLSubmissionNegativeAcknowledgementMessage}
+import models.{ArrivalId, Arrivals, ResponseMessage}
+import play.api.Logger
 import play.api.http.HeaderNames
 import play.api.libs.ws.{WSClient, WSResponse}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpReads, HttpReadsTry}
+import uk.gov.hmrc.http.HttpReads.is2xx
+import uk.gov.hmrc.http.{HeaderCarrier, HttpReads, HttpReadsTry, HttpResponse}
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.xml.NodeSeq
 
 class ArrivalMovementConnector @Inject()(config: FrontendAppConfig, http: HttpClient, ws: WSClient)(implicit ec: ExecutionContext) extends HttpReadsTry {
+  val logger: Logger = Logger(getClass)
```

Comment:
To make adding logging easier, we've adopted [this pattern](https://github.com/hmrc/declare-transit-movement-arrival-frontend/blob/master/app/logging/Logging.scala), that way the loggers have a predictable pattern. I think that's worth adding to this repo